docker-compose.yml

```diff
@@ -219,6 +219,7 @@ services:
       context: ./cron
       args:
         BASE_IMAGE: "$SENTRY_IMAGE"
+    entrypoint: "/entrypoint.sh"
```

Comment:
Don't we need this for all cron-ish images?

Reply:
Nope because we don't override any other `entrypoint`s yet. Once that happens, we'll have a `<<: cron_common` or something. We can add that as a precaution but then we'd need to add a note to the `Dockerfile` that we are overriding the `ENTRYPOINT` command anyway.

Reply:
I.e., also `snuba-cleanup` and `symbolicator-cleanup`.

Reply:
> we don't override any other `entrypoint`s yet

Meaning in `SNUBA_IMAGE` and `SYMBOLICATOR_IMAGE`, yes?

Reply:
I see now, `snuba-cleanup` inherits from `snuba_defaults`/`x-snuba-defaults`, which inherits from `restart_policy`/`x-restart-policy`, which doesn't have the `entrypoint` override that `x-sentry-defaults` has. Similarly with `symbolicator-cleanup`, which inherits from `restart_policy`/`x-restart-policy` directly. 👍